DVCodingStyle.md

```diff
@@ -1074,7 +1074,10 @@ assert(function_that_has_side_effects());
 Always put `wait fork` and `disable fork` constructs inside of an
 `isolation fork...join` block to avoid erroneous waiting.
 It is recommended to use the `DV_SPINWAIT` macro whenever possible,
-[as described here](#macro-usage)
+[as described here](#macro-usage).
```

Comment:
nit: Seems like this file is using the 80 char per line 'README' file convention.

Reply:
Should be fixed now.